### PR TITLE
Custom web ui chapter - step 1 and step 2

### PR DESCRIPTION
**Pull request by TiagoBrenck**
## Purpose
Included the phase 1 for custom web UI (customizing the HTML) and included its link in the root readme.

Included the phase  2 for custom web UI (custom browser).

Note: The code PR was done separately here: https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/8

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            Console.WriteLine($"Hello {result.Account.Username}");
+            Console.Write(Environment.NewLine);
+            Console.WriteLine("-------- GRAPH RESULT --------");
+            Console.Write(Environment.NewLine);
+            Console.WriteLine($"Id: {me.Id}");
+            Console.WriteLine($"Display Name: {me.DisplayName}");
+            Console.WriteLine($"Email: {me.Mail}");
+        }
+
+        private static GraphServiceClient GetGraphServiceClient(string accessToken, string graphApiUrl)
```
Comment (jmprieur) [inline 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs]: Don't we want to use the pattern where the delegate does the try/catch ?
See https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/9
cc: @Shama-K @kalyankrishna1

File: 3-CustomWebUI/3-1-CustomHTML/Console-Interactive-CustomWebUI/Program.cs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            Console.WriteLine("-------- GRAPH RESULT --------");
+            Console.Write(Environment.NewLine);
+            Console.WriteLine($"Id: {me.Id}");
+            Console.WriteLine($"Display Name: {me.DisplayName}");
+            Console.WriteLine($"Email: {me.Mail}");
+
+            Console.ReadKey();
+        }
+
+        private static GraphServiceClient GetGraphServiceClient(string accessToken, string graphApiUrl)
```
Comment (jmprieur) [inline 3-CustomWebUI/3-1-CustomHTML/Console-Interactive-CustomWebUI/Program.cs]: Could we use the pattern where the delegate does the try/catch?
like in https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/9

**Review by jmprieur — COMMENTED** [review]
Thanks @TiagoBrenck 
We're getting there.
I suggested to use the pattern where the Graph calls a delegate that does the try catch

like what @Shama-K has done in 
https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/pull/9

Thanks!

File: 3-CustomWebUI/3-1-CustomHTML/Console-Interactive-CustomWebUI/Program.cs (hunk 2 above)
Comment (TiagoBrenck) [reply 3-CustomWebUI/3-1-CustomHTML/Console-Interactive-CustomWebUI/Program.cs]: Ok, I will wait for Shama s PR to get merged and update my code.

File: 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs (hunk 1 above)
Comment (TiagoBrenck) [reply 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs]: I changed the sample to use Shama s code @jmprieur

**Comment by TiagoBrenck** [comment]
@jmprieur I updated the code to use the new Graph pattern

File: 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            {
+                // Acquiring an access token interactively. MSAL will cache it so we can use AcquireTokenSilent
+                // on future calls.
+                result = await app.AcquireTokenInteractive(scopes)
+                            .ExecuteAsync();
+            }
+
+            string graphApiUrl = configuration.GetValue<string>("GraphApiUrl");
+            // Instantiating GraphServiceClient and using the access token acquired above.
+            var graphClient = GetGraphServiceClient(result.AccessToken, graphApiUrl);
```
Comment (jmprieur) [inline 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs]: @TiagoBrenck you missed this one?

**Review by jmprieur — COMMENTED** [review]
LGTM.
I think you missed one, @TiagoBrenck

File: 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs (hunk 3 above)
Comment (TiagoBrenck) [reply 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs]: Oh, 3-2 is actually not even ready. Thanks for pointing that

Comment (TiagoBrenck) [reply 3-CustomWebUI/3-2-CustomBrowser/Console-Interactive-CustomWebUI/Program.cs]: Updated @jmprieur

**Review by jmprieur — APPROVED** [review]
LGTM. 
Thanks @TiagoBrenck 
Awesome work!

**Pull request merged**